test/Microsoft.OpenApi.Trimming.Tests/Microsoft.OpenApi.Trimming.Tests.csproj

```diff
@@ -4,17 +4,19 @@
     <TargetFramework>net8.0</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
+    <PublishTrimmed>true</PublishTrimmed>
     <PublishAot>true</PublishAot>
     <TrimmerSingleWarn>false</TrimmerSingleWarn>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-    <NoWarn>NU1903</NoWarn>
+    <NoWarn>NU1903; IL3000</NoWarn>
```

Comment:
I'd support an in place suppression instead to avoid future changes introducing similar issues without us noticing.

Reply:
There's no code line attached to the warning as it's caused by external code:
https://github.com/microsoft/OpenAPI.NET/actions/runs/12812328824/job/35723707783?pr=2063

We've just replicated the dotnet sdk's implementation here:
https://github.com/microsoftgraph/msgraph-sdk-dotnet-core/blob/c225b2e7d291de95ca28b9d5529b276d63019dd4/tests/Microsoft.Graph.DotnetCore.Core.Trimming/Microsoft.Graph.DotnetCore.Core.Trimming.csproj#L12